### PR TITLE
Add dependency fix for sudden BigQuery test failure

### PR DESCRIPTION
**Pull request by vinceatbluelabs**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -105,6 +105,19 @@ def initialize_options(self) -> None:
     #
     # https://github.com/googleapis/google-auth-library-python/issues/190
     'google-api-python-client>=1.8.0,<1.9.0',
+    #
```
Comment (bluelabsbutler) [inline setup.py:108]: This file, one of the 3 largest in the project, increased in size to 357 lines.  The total size of those files is now 3399 lines (target: 1015).  Is this file complex enough to refactor?

Comment (bluelabsbutler) [inline setup.py:108]: This file, one of the 3 largest in the project, increased in size to 357 lines.  The total size of those files is now 3399 lines (target: 1028).  Is this file complex enough to refactor?

Comment (vinceatbluelabs) [reply setup.py:108]: Unfortunately we established you really can't break up stuff in setup.py without breaking some different packaging scenarios.

File: setup.py [hunk 2]
```diff
@@ -105,6 +105,19 @@ def initialize_options(self) -> None:
...
+    # For some reason this issue started happening consistently around 2020-10:
+    #
+    #  https://app.circleci.com/pipelines/github/bluelabsio/records-mover/1134/workflows/267ca651-def1-4f60-bdfb-0f857a9e0c60/jobs/9770
+    #
+    # This seems to be a clue:
+    #
+    #  https://github.com/pypa/pip/issues/8407
```
Comment (cwegrzyn) [inline setup.py:115]: Praying for the new resolution stuff to be out and stable soon-- although i'm sure it's going to break all sorts of back/forwards compatibility things, sigh.

**Review by cwegrzyn — COMMENTED** [review]
:shipit:

**Pull request merged**